docs/manuals/gui/viselements/charts/filled-area-plots.md

```diff
@@ -0,0 +1,155 @@
+## Filled Area Plots
+
+An area chart displays a solid color between the traces of a graph. It is an option on scatter charts. Areas will be filled according to the ‘fill’ option ("none", "tozeroy", "tozerox" , "tonexty", "tonextx", "toself", and "tonext")
+
+### Simple example
+This example shows the number of sold items per day (The shop closes on Weekend :) ).
+```py
+
+from taipy.gui import Gui
+import pandas as pd
+ 
+options = {"fill":"tonextx"}
+ 
+data = {
+  "day": ['Mon', 'Tue','Wed','Thu','Fri'],
+  "items": [32, 25, 86, 60, 70],
+ }
+ 
+ 
+md = "<|{data}|chart|x=day|y=items|type=scatter|options={options}|>"
+ 
+Gui(md).run()
+```
+![Simple example](filled-area.png)
+
+### Basic Overlaid Area Chart
+With above example, we want to add one more trace to visualize the total price (x 1000$) per day.
```

Comment:
=>
You can overlay two different data sets in the same chart. Each data set will be represented by its own plot.

Starting from the example above, we can add the price of the items: